todo/src/components/TodoForm.js

```diff
@@ -0,0 +1,39 @@
+import React, { useState } from 'react'
+import { connect } from 'react-redux'
+import { addTodo } from '../actions'
+
+import { FormContainer, Form } from './S_TodoForm'
+
+const TodoForm = props => {
+
+  const [value, setValue] = useState('')
+
+
+  const handleChanges = e => {
+    setValue(e.target.value)
+  }
+
+  const submitTodo = e => {
+    e.preventDefault();
+    props.addTodo(value)
+
+    setValue('')
+  }
+
+  return (
+    <FormContainer>
+      <Form onSubmit={submitTodo}>
+        <input
+          name='newTodo'
+          value={value}
+          type='text'
+          placeholder='Enter New Todo'
+          onChange={(e) => handleChanges(e)}
```

Comment:
Another way to do this is:
```
  onChange={e => setValue(e.target.value)}
```
This way, you don't even need to define the handleChanges method. Less code to write, and cleaner.